extensions/amp-loader/0.1/amp-loader.js

```diff
@@ -47,6 +46,39 @@ const LOADER_BACKGROUND_TAGS = {
   'AMP-GOOGLE-DOCUMENT-EMBED': true,
 };
 
+/**
+ * All video player components must either have a) "video" or b) "player" in
+ * their name.
+ * @private @const {!RegExp}
+ */
+const VIDEO_PLAYER_TAG_PATTERN = /^amp\-(video|.+player)/;
+
+/**
+ * A few components don't follow the previous convention for historical
+ * reasons, so they are listed individually.
+ * @private @enum {boolean}
+ */
+const VIDEO_PLAYER_TAGS_LEGACY = {
+  'AMP-BRIGHTCOVE': true,
+  'AMP-DAILYMOTION': true,
+  'AMP-YOUTUBE': true,
+  'AMP-VIMEO': true,
+  'AMP-IMA-VIDEO': true,
+};
```

Comment:
I'm not a fan of the `amp-loader` extension containing this list nor `LOADER_BACKGROUND_TAGS` above.

https://github.com/ampproject/amphtml/blob/58eddda4a07ab8bab2112071845c87c2986953d5/extensions/amp-loader/0.1/amp-loader.js#L34-L47

However, these lists are used in a more subtle manner than the logic to display loaders in `v0`, so I think it's best to optimize those separately.